airbyte-server/src/main/java/io/airbyte/server/converters/ConfigFileArchiver.java

```diff
@@ -94,9 +96,9 @@ private Path buildConfigPath(ConfigSchema schemaType) {
         validateJson(config, schemaType);
         results.add(config);
       }
-      LOGGER.debug(String.format("Successful read of airbyte config %s", schemaType));
+      LOGGER.debug(String.format("Successful read of airbyte config %s from archive", schemaType));
     } else {
-      LOGGER.debug(String.format("Airbyte config %s was not found", schemaType));
```

Comment:
you don't think it would be helpful to let people choose if they want to migrate/import partially the configurations vs the whole package along with the DB too?

For example, I want to share my airbyte setup with you but you probably don't want to overwrite your workspace config file with mine (then my email/id will be the contact on your instance)

Reply:
> you don't think it would be helpful to let people choose if they want to migrate/import partially the configurations vs the whole package along with the DB too?

no. there's no use case for that right now in imo. it's just going to lead to a totally broken state that someone is going to ask us to debug. worse they might not realize what has happened and their configuration will be corrupted and they will have to start from scratch. let's get the product working as expected before we go for this level of configuration.